WireAnalytics/Sources/WireAnalytics/Countly+AnalyticsService.swift

```diff
@@ -39,8 +39,8 @@ extension Countly: AnalyticsService {
         }
     }
 
-    public func trackEvent(_ event: AnalyticEvent) {
-        recordEvent(event.rawValue)
+    public func trackEvent(_ event: AnalyticsEvent) {
+        recordEvent(event.eventName)
```

Comment:
Do you also want to include the segmentation?

Reply:
Done: https://github.com/wireapp/wire-ios/commit/44c3b9deeafb41952c99293d14fbb3c9b817232b